docs/DeveloperGuide.adoc

```diff
@@ -295,35 +528,291 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` | event organizer | edit the weight of a lifting attempt for an athlete |
+so that the lifting order of athletes can be updated
+
+|`* * *` | event organizer | update an athlete’s score after a successful lift |
+so that the ranking of the athletes can be updated
+
+|`* * *` | event organizer | edit an athlete's name |
+update the athlete's name in case the athletes have change their names or they were inputted incorrectly previously
+
+|`* * *` | event organizer | input/edit an athlete's body weight |
+update the athlete's weight so the athelete is taking part in the correct weight class
+
+|`* * *` | event organizer | record when a lifter fails/succeed in his/her lift |
+update the athlete's score and ranking accordingly and to keep a record for how each athlete's
+score was derived
+
+|`* *` |event organizer | input/edit an athlete's country/club |
+which country/club the athlete is representing
+
+|`* * *` |event organizer | know the rack height for bench press/squat |
+the racks can be prepared before the athletes make their lifting attempt and save time
 
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
+|`*` | event organizer |find which club/country an athlete is affiliated with |
+so that I know which countries/clubs can be ranked across the various weight classess
+
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
-== Use Cases
+== Use Cases (To Do)
+
+(For all use cases below, the *System* is the `Powerlifting Competition Competitor` and the *Actor* is the `user`, unless specified otherwise)
+
+=== Out of competition mode
+
+[discrete]
+===== Use case: UC1 - View my competition list
+
+*MSS*
+
+1.  User requests to list all competitions in the system.
+2.  System shows a list of competitions held based on the files in the folder.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty.
++
+Use case ends.
+
+[discrete]
+==== Use case: UC2 - Add a new competition
+
+*MSS*
+
+1. User requests for a new powerlifting competition to be started.
+2. User inputs the initial competition data (e.g. name, particpants) to be inputted.
+3. System adds the new competition to the user's competition list.
+4. System shows a success message.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. Competition data is invalid.
++
+[none]
+** 2a1. System shows an error message.
++
+Use case resume at step 2.
+
+* 2b. The competition name has already been used in user's competition list.
++
+[none]
+** 2b1. System tells user that the competition is already in his competition list.
++
+Use case ends.
+
+[discrete]
+==== Use case: UC3 - Delete a competition
+
+*MSS*
+
+1. User obtains a list of competitions by listing (<<Use case: UC1 - View my competition list, *UC1*>>)
```

Comment:
Underline the listing with the UC1 since it is referenced. Also, whenever you reference a use case, the same name has to be used